src/redux/reducers/userReducer.ts

```diff
@@ -24,10 +32,12 @@ export const userReducer = (state: User | null = null, action: UserActions | Orc
         affiliations,
         givenName: action.user.given_name,
         familyName: action.user.family_name,
-        isPublisher: roles.some((role) => role === RoleName.PUBLISHER),
-        isAppAdmin: roles.some((role) => role === RoleName.APP_ADMIN) || action.user.email.endsWith('@unit.no'), // TODO: temporarily set app admin role based on email
-        isInstitutionAdmin: roles.some((role) => role === RoleName.ADMIN),
-        isCurator: roles.some((role) => role === RoleName.CURATOR),
+        isPublisher: roles ? roles.some((role) => role === RoleName.PUBLISHER) : false,
+        isAppAdmin: roles
+          ? roles.some((role) => role === RoleName.APP_ADMIN) || action.user.email.endsWith('@unit.no')
+          : false, // TODO: temporarily set app admin role based on email
+        isInstitutionAdmin: roles ? roles.some((role) => role === RoleName.ADMIN) : false,
+        isCurator: roles ? roles.some((role) => role === RoleName.CURATOR) : false,
         possibleAuthorities: [],
       };
```

Comment:
hva er egentlig nytt/fiksen her? for meg virker det som det er mer kode som gjør det samme

Reply:
fikk denne erroren:
blob:https://unit.atlassian.net/6ea1b287-0518-4f79-a336-381bd5c7b6e2

Reply:
men vi kan vel ikke gjøre `replace` på et tomt array heller?

Reply:
haha, det var den fantastiske array-strengen som ikke kan parses til array det ja :D 
vil egentlig bare slippe alle null-sjekkene, så vi kunne kanskje gjort noe sånn også? 
```
const feideAffiliations = action.user['custom:affiliation'];
const feideRoles = action.user['custom:applicationRoles'];

const affiliations = feideAffiliations ? (...) : [];
const roles = feideRoles ? (...) : [];
```

Reply:
(savner auto-update av nye kommentarer i github)